### PR TITLE
added bazel recipe

### PR DESCRIPTION
**Pull request by ThomasWollmann**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by ThomasWollmann** [comment]
@bgruening do you know how to enforce .sh execution via bash? Seems like chmod does not work.

**Comment by bgruening** [comment]
Can it be that this file is generated on the fly and had no chmod`ing before?

**Comment by ThomasWollmann** [comment]
maybe ... what is the solution?

**Comment by bgruening** [comment]
patching the compile.sh script?

**Comment by ThomasWollmann** [comment]
Seems a strange solution, since on other systems it compiles flawlessly.

**Comment by bgruening** [comment]
@ThomasWollmann what are these other systems? You compile here on a CentOS-6 and a OSX. Both normal systems I would say.
I would need to look deeper into it, maybe the error is very hidden.

**Comment by ThomasWollmann** [comment]
http://blog.abysm.org/2016/06/building-tensorflow-centos-6/

File: recipes/bazel/build.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-chmod u+x **/*.sh
+scl enable devtoolset-4 bash
```
Comment (bgruening) [inline recipes/bazel/build.sh]: This is not allowed. If you want to have this, remove the dependency on gcc from your recipe.

Comment (ThomasWollmann) [inline recipes/bazel/build.sh]: I guess this will cause problems with glic.

**Comment by ThomasWollmann** [comment]
https://anaconda.org/conda-forge/bazel

**Comment by bgruening** [comment]
:)

**Pull request closed without merging**